tests/core/fixtures.py

```diff
@@ -112,13 +123,52 @@ def chain(chaindb):
 
 
 @pytest.fixture
-def shard_chain():
-    shard_chaindb = BaseChainDB(get_db_backend(), account_state_class=ShardingAccountStateDB)
-    return chain(shard_chaindb)
+def shard_chain(shard_chaindb):
+    shard_chaindb = shard_chaindb
+    genesis_params = {
+        "bloom": 0,
+        "coinbase": to_canonical_address("8888f1f195afa192cfee860698584c030f4c9db1"),
+        "difficulty": 131072,
+        "extra_data": b"B",
+        "gas_limit": 3141592,
+        "gas_used": 0,
+        "mix_hash": decode_hex(
+            "56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421"),
+        "nonce": decode_hex("0102030405060708"),
+        "block_number": 0,
+        "parent_hash": decode_hex(
+            "0000000000000000000000000000000000000000000000000000000000000000"),
```

Comment:
*nitpick*

These lines are probably easier to read if you keep it on one line and add a `# noqa: E501` to the end to whitelist the long line for the linter.